### DISCUSSION TITLE
would it be better to use "disabled" here?

### DISCUSSION THREAD

**Discussion by DuckSoft** [shadowsocks/shadowsocks-rust#402] [Ideas]
ref: https://github.com/shadowsocks/shadowsocks-rust/pull/401

**Comment by database64128** [comment]
Since the change was just merged hours ago, we can just change it to `disabled`.

**Comment by zonyitoo** [comment]
What do you think? @spyophobia

**Comment by spyophobia** [comment]
🤔
I don't really see a difference but sure I guess.

**Comment by zonyitoo** [comment]
done.

**Comment by spyophobia** [comment]
Don't forget about README
